euclidean_gcd/CPP/euclidean_gcd.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+int gcd_rec(int a, int b)
+{
+	return b == 0 ? a : gcd_rec(b, a % b);
+}
+int gcd_loop(int a , int b) {
+	while (b != 0)
+	{
+		int temp = b;
+		b = a % b;
+		a = temp;
```

Comment:
{ should almost always be at the end of the previous line  [whitespace/braces] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*